demo/package.json

```
@@ -10,7 +10,7 @@
     "type-check": "tsc --noEmit"
   },
   "dependencies": {
-    "@stoplight/elements": "*",
+    "@stoplight/elements": "7.0.0-beta.1",
```

Would `7.0.0@beta` make this less rigid so we dont have to do updates when new betas are out? 
```suggestion
    "@stoplight/elements": "7.0.0@beta",
```

---

I'm intentionally making it more rigid. It's part of the lerna workspace, so whenever we release a new version, lerna will auto-bump this dependency, too. No manual updates needed.:)